apps/platform-app/src/app/[locale]/(profile)/(components)/CVuploader/CvUploader.tsx

```diff
@@ -0,0 +1,12 @@
+import { CVuploaderForm } from "./CvUploaderForm"
+import styles from './CvUploader.module.scss'
+
+export const CVUploader = () => {
+  return (
+    <div className={styles.container}>
+      <p className={styles.containerLabel}>CV file (optional)</p>
+      <CVuploaderForm profileId="" />
```

Comment:
**suggestion**: Consider passing a meaningful 'profileId' instead of an empty string to avoid potential issues.